### PR TITLE
fix building

### PR DESCRIPTION
**Pull request by terminal-cs**

### CODE REVIEW

**Comment by Kiirx** [comment]
I tought I fixed this lol, ill try again since I just got on Debian 11

File: source/Cosmos.Debug.Hosts/Qemu.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿/*
+using System;
```
Comment (quajak) [inline source/Cosmos.Debug.Hosts/Qemu.cs:2]: Can you find a fix which doesnt require commenting out all of Qemu? Or is the file no longer needed?

**Review by quajak — CHANGES_REQUESTED** [review]

File: source/Cosmos.Debug.Hosts/Qemu.cs (hunk 1 above)
Comment (zarlo) [reply source/Cosmos.Debug.Hosts/Qemu.cs:2]: its used in the current master

Comment (terminal-cs) [reply source/Cosmos.Debug.Hosts/Qemu.cs:2]: We can just un-comment the whole file when it comes time to merge

**Pull request closed without merging**